### PR TITLE
Higher-frequency checking with alerts

### PR DESCRIPTION
**Pull request by jeanconn**
Add changes to support use as an hourly checker for replan central.

### CODE REVIEW

**Comment by jeanconn** [comment]
I think this is runnable in this state and I'll run it on the side now.  If you don't like the H5 checks, they can each be changed to a FileWatch.  If you don't like the iFOT checks, they can obviously be cut.

File: jobwatch.py [hunk 1]
```diff
@@ -261,10 +280,12 @@ def remove_old_reports(rootdir, date_now, max_age):
             shutil.rmtree(outdir)
 
 
-def sendmail(recipients, html, datenow):
+def sendmail(recipients, html, datenow, subject=None):
+    if subject == None:
```
Comment (taldcroft) [inline jobwatch.py]: Nit: `if subject is None`

**Comment by taldcroft** [comment]
Looks great!  The only thing I'm really not happy with is having skawatch classes be in days and the rest be in hours.  I think the original unit of days is generally more appropriate.  So for instance in arcwatch you could just keep everything in days but define a constant `HOUR` and specify things in multiples of that.

Also I'd be happier if arcwatch.py was named something more generic since there isn't anything in the code or design that is specific to arc files.

**Comment by jeanconn** [comment]
Sure.   And setting the fundamental time unit to be the smaller one made
more sense to me,  but I suppose numeric stability is not an issue here.

**Comment by taldcroft** [comment]
We really should be using astropy quantities and Time anyway.  Then any unit is fine.  When 1.0 comes out we need to upgrade.  Astropy.time is a lot better than Chandra.Time in every way.

**Comment by jeanconn** [comment]
I couldn't think of other stuff we'd add to the checking to figure out
another common denominator wrt a name,  though I went for
hourly_template.html, so I suppose it could be hourly _watch.  That seems
worse.

Whatever we call it, should it end up with its own task_schedule and such?
What makes the most sense to you for how to run it?

**Comment by taldcroft** [comment]
I was thinking about `hourly_watch` as well.  That actually conveys a lot of meaning that `arcwatch` would not, especially for anyone else.  In all likelihood it will really be an hourly watch, but even if it drifts in frequency that wouldn't bother me too much.  It's like how "weekly schedule" isn't really weekly, but it conveys a certain meaning.

**Comment by taldcroft** [comment]
One way to think about it is that, because you insisted on doing things correctly (:smile:), we are really checking the upstream products as well as arc (at least for web products with correct header info).  So the scope is beyond just arc because we might report problems to MTA or SWPC.

**Comment by taldcroft** [comment]
Yes, this will probably need its own task_schedule and a separated task directory for the logs.  I'm trying to remember if just setting the `task` in the config file to something else is sufficient.  It might make sense to set this up as a persistent running job like arc.

**Comment by jeanconn** [comment]
Though as far as I can tell the only real benefit to a persistent job is
that the task log rotation works better.   We might not care about that for
this task.

**Comment by taldcroft** [comment]
I think the other benefit is that you can have the checker cron job running on different machines for reliability, with only one instance getting the chance to actually run.

**Comment by jeanconn** [comment]
I think that could also be accomplished by setting up the cron start times
to work reasonably with the heartbeat timeout,  but maybe I'm missing a
subtlety.

**Comment by taldcroft** [comment]
That would be an OK workaround if we didn't already have the capability
built in to task schedule...

**Comment by jeanconn** [comment]
Isn't that how we use the capability in task_schedule even for the
persistent case? (offset cron jobs start times, reasonable heartbeat
value)?  ​

**Comment by taldcroft** [comment]
In the persistent case there's no fine tuning with the timeout because task
schedule touches the heartbeat file every minute.

**Comment by jeanconn** [comment]
​Ah.  Thanks.  I'd forgotten that.

**Comment by jeanconn** [comment]
I think the hourly_watch portion of this is safe and effective.  Do you have any concerns about the changes in jobwatch.py designed largely to make it easier to make a single status page?  Otherwise, I think this PR can go.

**Comment by taldcroft** [comment]
:+1:

**Pull request merged**